CMakeLists.txt

```diff
@@ -322,20 +307,21 @@ set(AI_LIBS
         PRIVATE roboteam_utils
         PRIVATE roboteam_proto
         PRIVATE rtt::bt
+        PRIVATE rtt::control
         PRIVATE ${SDL2_LIBRARY}
         )
 
 set(AI_INCLUDES
         INTERFACE include
-        PRIVATE include/roboteam_ai
+        PUBLIC include/roboteam_ai
```

Comment:
wtf, no. This shouldnt be necessary.

Reply:
I changed that because the includes wouldn't work in submodule, will revert, good catch